sdk/src/app_data.rs

```diff
@@ -0,0 +1,38 @@
+use std::{
+    any::{Any, TypeId},
+    fmt,
+};
+
+use ahash::AHashMap;
```

Comment:
I think we can just use a normal hash map here

Reply:
Alternatively, https://crates.io/crates/typemap

Reply:
Typemap is pretty cool, didn't know about it.  It requires a "Key" trait implementation in order to insert and retrieve though.  Do you want to expose this trait as a "pub use" from the rust SDK or ask people to import the crate themselves?  I'm going to remove the ahash crate first and use a normal hashmap first while we decide about this.